definitions/Campaign.dbd

```diff
@@ -10,7 +10,7 @@ int<PlayerCondition::ID> Completed
 int<UiQuestDetailsTheme::ID> UiQuestDetailsThemeID
 int Flags // &0x01: useNormalQuestIcons
 int<PlayerCondition::ID> OnlyStallIf // if given and evals to false, don't mark as stalled.
-int<PlayerCondition::ID> Field_9_0_1_35755_007?
+int<PlayerCondition::ID> PlayerConditionID?
```

Comment:
I had intentionally not named this one since there are multiple conditions in the table and they all have different semantics. I don’t know the semantic of this one, thus the missing name.

Reply:
Alright, will change it, thanks